### DISCUSSION TITLE
群聊发送消息失败，但单聊正常

### DISCUSSION THREAD

**Discussion by heavenscar** [Arondight/Adachi-BOT#871] [Q&A]
0|bot      | [2022-08-02T01:14:56.249] [ERROR] [iPad:xxxxx] - failed to send: [Group: xxxxx] undefined(46)
0|bot      | [2022-08-02T01:14:56.250] [ERROR] [iPad:xxxxx] - 错误：消息发送失败，因为“unknown”。
0|bot      | [2022-08-02T01:14:56.261] [ERROR] [iPad:xxxxx] - failed to send: [Group: xxxxx] undefined(46)
0|bot      | [2022-08-02T01:14:56.261] [ERROR] [iPad:xxxxx] - 错误：消息发送失败，因为“unknown”。

**Comment by Arondight** [comment]
#858 是不是这个问题
